src/pages/[platform]/build-a-backend/data/custom-business-logic/connect-amazon-translate/index.mdx

```diff
@@ -0,0 +1,256 @@
+import { getCustomStaticPath } from '@/utils/getCustomStaticPath';
+
+export const meta = {
+  title: 'Connect to Amazon Translate for language translation APIs',
+  description:
+    'Connect to Amazon Translate.',
+  platforms: [
+    'android',
+    'angular',
+    'flutter',
+    'javascript',
+    'nextjs',
+    'react',
+    'react-native',
+    'swift',
+    'vue'
+  ]
+};
+
+export const getStaticPaths = async () => {
+  return getCustomStaticPath(meta.platforms);
+};
+
+export function getStaticProps() {
+  return {
+    props: {
+      meta
+    }
+  };
+}
+
+# Connect to Amazon Translate for language translation APIs
```

Comment:
Don't think we need this extra heading here. On the page it shows as double because the docs site will automatically add the `meta.title` as the heading

This is what it looks like on my localhost:
![image](https://github.com/aws-amplify/docs/assets/54393192/02ec4816-23e3-49ca-9b02-a33508f190e2)

Reply:
removed extra title